editor/edit-post/sidebar/header.js

```diff
@@ -32,9 +32,9 @@ const SidebarHeader = ( { panel, onSetPanel, onToggleSidebar, count } ) => {
 			<button
 				onClick={ () => onSetPanel( 'block' ) }
 				className={ `editor-sidebar__panel-tab ${ panel === 'block' ? 'is-active' : '' }` }
-				aria-label={ __( 'Block settings' ) }
+				aria-label={ __( 'Format settings' ) }
 			>
-				{ sprintf( _n( 'Block', '%d Blocks', count ), count ) }
+				{ __( 'Format' ) }
```

Comment:
We'd still want the count to be used, just not the Block(s) variation.

Reply:
I think I've done that now.